src/search/DownwardFiles.cmake

```diff
@@ -305,30 +305,45 @@ fast_downward_plugin(
 )
 
 fast_downward_plugin(
+<<<<<<< HEAD
+    NAME SUB_EVALUATOR
+    HELP "Plugin for WA, XDP, XUP & PWXDP priority functions"
+    SOURCES
+        evaluators/sub_evaluator
+    DEPENDS EVALUATORS_PLUGIN_GROUP
+)
+
+fast_downward_plugin(
+    NAME XDP_EVALUATOR
+=======
     NAME XDP
+>>>>>>> ecea56b08... Add functional XDP
```

Comment:
It seems that merging this file was aborted. I'm not sure how this compiles at the moment :-)